substrate/mmap

```diff
@@ -126,18 +168,17 @@ namespace substrate
 		~mmap_t() noexcept
 		{
 			if (_addr)
-				::munmap(_addr, _len);
+				unmap(_addr);
 			if (_fd != -1)
 				::close(_fd);
 		}
 
 		SUBSTRATE_NO_DISCARD(SUBSTRATE_CXX14_CONSTEXPR bool valid() const noexcept) { return _addr; }
 #else
-		mmap_t(const fd_t &fd, const std::size_t len, const DWORD prot, const int32_t = 0,
+		mmap_t(const int32_t fd, const std::size_t len, const int32_t prot, const int32_t = 0,
```

Comment:
Please validate here if I didn't do a dumdum, because Linux expects a `int32_t` for `cleanProt` while Windows needs a `DWORD`.

Reply:
Fair call - you suspected correctly, we'll amend the commit.